homeassistant/components/camera/local_file.py

```diff
@@ -11,31 +11,44 @@
 import voluptuous as vol
 
 from homeassistant.const import CONF_NAME
-from homeassistant.components.camera import Camera, PLATFORM_SCHEMA
+from homeassistant.components.camera import (
+    Camera, CAMERA_SERVICE_SCHEMA, DOMAIN, PLATFORM_SCHEMA)
 from homeassistant.helpers import config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_FILE_PATH = 'file_path'
-
 DEFAULT_NAME = 'Local File'
+SERVICE_UPDATE_FILE_PATH = 'update_file_path'
```

Comment:
As Paulus said, platform-specific services need to be prefixed by the platform name. So in this case `local_file_update_file_path` (I prefer this one, it's longer but consistent with the configuration schéma) or `local_file_update_path`.

Reply:
ok sure